lte/gateway/python/magma/pipelined/app/arp.py

```diff
@@ -165,7 +165,7 @@ def handle_restart(self,
 
         for rec in records:
             if rec.id not in attached_ues and \
-               rec.id.replace('imsi', '') not in attached_ues:
+               rec.id.replace('IMSI', '') not in attached_ues:
```

Comment:
Might be cleaner to move this method to Imsi.py to add and remove 'IMSI' or 'imsi' and use that everywhere.

Reply:
I don't think we do any other IMSI comparison in pipelined